### ISSUE TITLE
Inconsistent references

### ISSUE THREAD

**Issue by adnanvaldes** [zenodotus280/apl-md#14]
The citation at the end of each pattern is inconsistent. For example, "Agricultural Valleys (4)" has a full citation written inside a ">[!cite]" callout (and also includes tags, such as `> #APL/Towns/Regional-Policies`):

```
---
> [!cite]- Alexander, Christopher. _A Pattern Language: Towns, Buildings, Construction_. Oxford University Press, 1977, p. 26.
> #medium-confidence 
> 
> #APL/Towns/Regional-Policies 
```

Other patterns, for example "Alcoves (179)" simply include a line of text without a callout or tags:

```
---
Reference for full-text of Pattern: p. 828 #high-confidence 
```

I am happy to fix the issue to ensure consistent styling with the callouts. I imagine the best way to do it would be to have the `>[!cite]` block for every note in the same format as "Agricultural Valleys", including having the confidence tag on a new line.

Regarding the other tag I am not sure how to handle that, since I don't (unfortunately) own a copy of the book itself.

**Comment by zenodotus280** [comment]
Yes, this is where I started to slow down and decided to push the notes out the door. I'll create a branch for this and I can start adding the updated tags once you have search-replaced the existing text into the new reference format.

**Comment by zenodotus280** [comment]
Leaving open for now until I update the tags and white space differences.

**Comment by adnanvaldes** [comment]
Could you give me an example of the white space inconsistencies that you mention in the merge comments? As it stands, with the PR every note has the following format:

```
...end of text...

---
> [!cite]- Alexander, Christopher. _A Pattern Language: Towns, Buildings, Construction_. Oxford University Press, 1977, p. XXX.
> #xxx-confidence
```

The few notes that have tags have the extra line after confidence and then the tag itself:

```
...end of text...

---
> [!cite]- Alexander, Christopher. _A Pattern Language: Towns, Buildings, Construction_. Oxford University Press, 1977, p. XXX.
> #xxx-confidence
>
> Tags
```

So are you referring to white space inconsistencies _immediately_ after the `---` line? Or to white space after the `#confidence` line?

**Comment by zenodotus280** [comment]
I am referring to the line space after "---" which should be present so that a full horizontal bar will reliably render in Obsidian or other markdown tools. It looks like it was removed in error. It's incredibly minor and mostly a personal style preference which I don't want you to waste your time on ;)

---

[Original with space](https://raw.githubusercontent.com/zenodotus280/apl-md/refs/heads/master/Patterns/Magic%20of%20the%20City%20(10).md)
```
---

> [!cite]- [...]
```


[PR without space](https://raw.githubusercontent.com/zenodotus280/apl-md/c132ff2e0b3b48e2de3968013b01c9b9a87ff15f/Patterns/Magic%20of%20the%20City%20(10).md)
```
---
> [!cite]- [...]
```

**Comment by adnanvaldes** [comment]
Ah, fair point! I didn't notice that when I was updating the files. Anyway, I've created a new pull request that addresses this.

**Comment by zenodotus280** [comment]
Fixed by 4f5f3a05141c48dcad26f12c1f5cc9f380ec60cc
Thanks! :+1:
